### PR TITLE
Replaces gibbing on being heretic sacrificed with being sent to brazil

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# Document the changes in your pull request

fun and intuitive gameplay feature
getting sacrificed by a heretic will revive you and send you to a void instead of gibbing you, causing several negative effects over the course of a few minutes then teleporting you back to the station to somewhere probably safe

# Wiki Documentation

probably undocumented

# Changelog

:cl:  
tweak: heretic sacrifices no longer gib people
rscadd: new brazil area for holding heretic sacrifices (this is temporary) (but not good for your health)
/:cl:

### CODE REVIEW

**Comment by maxion12345** [comment]
kinda feels hugbox ngl
you are being sacrificed to an eldrich  abomination for internet points you should explode

File: code/datums/status_effects/debuffs.dm [hunk 1]
```diff
@@ -1065,3 +1065,119 @@
...
+
+/datum/status_effect/heretic_sacrifice
+	id = "heretic_sac"
+	alert_type = null
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = 4 MINUTES
+	tick_interval = 15 SECONDS
+	var/list/unspooked_limbs = list(BODY_ZONE_R_ARM, BODY_ZONE_L_ARM, BODY_ZONE_R_LEG, BODY_ZONE_L_LEG)
+	var/list/all_limbs = list(BODY_ZONE_L_ARM, BODY_ZONE_R_ARM, BODY_ZONE_L_LEG, BODY_ZONE_R_LEG, BODY_ZONE_CHEST, BODY_ZONE_HEAD)
+	var/fuckywuckycounter = 0
```
Comment (Hopekz) [inline code/datums/status_effects/debuffs.dm]: Imagine not adding a comment to what fuckywuckycounter is used for

**Comment by Cartlord** [comment]
This seems like a horrible idea

File: code/datums/status_effects/debuffs.dm (hunk 1 above)
Comment (SomeguyManperson) [reply code/datums/status_effects/debuffs.dm]: draft pr

**Comment by SomeguyManperson** [comment]
> kinda feels hugbox ngl you are being sacrificed to an eldrich abomination for internet points you should explode

you don't read many scary stories do you

**Comment by Xoxeyos** [comment]
Show me an interesting, and challenging environment, with some traps, and monsters thrown in, and I might be a bit more interested. I wouldn't want this to be an easy thing to come back from, and I'd want it to be an even more difficult thing to recover from.

File: code/game/turfs/closed.dm [hunk 2]
```diff
@@ -192,4 +192,4 @@
 	name = "wall"
 	desc = "A wall made out of a strange metal. The squares on it pulse in a predictable pattern."
 	icon = 'icons/turf/walls/hierophant_wall.dmi'
-	icon_state = "wall"
+	icon_state = "wall"
```
Comment (patpol4) [inline code/game/turfs/closed.dm]: :think:

**Review by patpol4 — APPROVED** [review]
Map appears fine.

**Comment by Marmio64** [comment]
Should make it like the scp 106 pocket dimension

**Comment by patpol4** [comment]
Sorry you had an extra space there, i couldn't stand it

**Comment by Hopekz** [comment]
IMO this PR has the potential to make this part much "cooler" instead of something simple like getting gibbed.
It might even introduce more fun into the round for the people sent, depending on how this void/"brazil" looks.

No idea why people have strong opinions for or against this yet since there isn't really much here to judge and the PR might look completely different once it is ready to be judged.

Also for @SomeguyManperson : Your changelog sucks so hopefully you make that better sometime

**Comment by Xoxeyos** [comment]
I know this is a draft, but putting a negative over a space, and leaving a 40x40~ area is unsatisfying, to say the least. 
[This is just a white space.](https://imgur.com/a/wEDEw6H)
I'd like to have to solve a puzzle of some sort, face difficult enemies, or have some other such obstacle.

**Comment by maxion12345** [comment]
A puzzle, or some choice would be infinitely better then just being sent somewhere for a bit then coming back. Makes getting captured have no real consequences for the one being captured, you just waste a bit of time. People are probably just going to ask people to sacrifice them (and people will do it) instead of going after them normally. We know there were people that would let vampires just suck their blood if they asked. Is that self-antaging? I don't know, kinda? But it still happens regardless.

**Comment by patpol4** [comment]
The japanese restaurant is full of content, yet people get bored to death there anyway.

**Comment by SomeguyManperson** [comment]
> The japanese restaurant is full of content, yet people get bored to death there anyway.

that's maybe because the ninja jail is somewhere you get put for the rest of the round and not for a couple minutes

**Comment by SomeguyManperson** [comment]
For reference, the current fun and intuitive heretic post-sacrifice gameplay looks something like this

image

you get gibbed and the majority of your stuff is deleted if it wasn't stripped first
the riveting gameplay afterwards is pressing the do not resuscitate button and waiting for a midround antag

**Comment by Xoxeyos** [comment]
I've already spoken quite a bit about my thoughts for this PR in the public dev channel, and I don't wish to repeat everything here because someone lacks the context for what I am saying, but I believe this can fleshed out much more; As I believe, putting someone in a blank space with a cheapo respawn timer on it's own isn't exactly inspiring.

https://discord.com/channels/134720091576205312/205784753352343552/903496910537629756

**Comment by Cartlord** [comment]
What the fuck were you thinking?
You closed your PR because of backlash, repoened it, and then merged it under 24 hours later. This stinks of cowardice, like you closed it to HIDE it from people until it was "ready" for a merge.

**Comment by maxion12345** [comment]
selfmerge go brrrr
also damn he is turning into oranges isn't he. We even got a photo of him preforming a lip to lip docking maneuver with a lizard

**Comment by Marmio64** [comment]
Kinda sus

**Comment by Hopekz** [comment]
Everyone is overreacting for no reason, this isn't "hidden" because everyone involved gets an email notification and it even gets posted to like 4 different places.

This change isn't significant enough to get mad over I'm pretty sure if it actually was a hidden commit that literally no-one would notice except a few weeks from now where someone would describe this as a "cool" thing that happens when you get sacrificed vs just getting killed.

**Comment by bruhlookatthisdood** [comment]
created a glitch where cult could summon someone who had been sent to brazil
resuliting in an invincible killing machine hailing nar'sir

**Pull request merged**